cxmon/src/mon_cmd.cpp

```diff
@@ -303,6 +307,256 @@ void binary_dump(void)
 
 
 /*
+ * Add Break Point
+ */
+void break_point_add(void)
+{
+	uintptr address;
+
+	if (mon_token == T_END || !mon_expression(&address)) {
+		mon_error("Expect break point in hexadecimal.");
+		return;
+	}
+
+	if (mon_token != T_END) {
+		mon_error("Too many arguments");
+		return;
+	}
+
+	BREAK_POINT_SET::iterator it;
+	// Save break point
+	if ((it = disabled_break_points.find(address)) == disabled_break_points.end())
+		active_break_points.insert(address);
+	else {
+		disabled_break_points.erase(it);
+		active_break_points.insert(address);
+	}
+}
+
+
+bool validate_index(uintptr *index_ptr, const BREAK_POINT_SET& break_point_set)
+{
+	if (mon_token == T_END || !mon_expression(index_ptr)) {
+		mon_error("Expect index number of break point in hexadecimal.\n");
+		return false;
+	}
+
+	if (mon_token != T_END) {
+		mon_error("Too many arguments");
+		return false;
+	}
+
+	if (*index_ptr > break_point_set.size()) {
+		mon_error("Illegal Index Number!");
```

Comment:
Nit: don't capitalize the last two words

Reply:
Done!